README.md

```diff
@@ -40,6 +40,11 @@ var res = md.render(src);
 console.log(res);
 ```
 
+## Options
+
+Enabling the `dataType` boolean flag in the options will generate figure tags that
+also declare the data-type being wrapped, e.g.: `<figure data-type="image">`. 
+This can be useful for applying special styling.
```

Comment:
Definitely other types, but I think this module exclusively handles images, doesn't it?

Difficult to say how common it is. I encountered the practice using the Webhook CMS (they do it with all inline content) and it seemed really smart to me.

Attrs are a good idea, but they seem somewhat contentious in the markdown / commonmark community, so I'm not sure about requiring them for this common a scenario. Your project though, so it's totally your call!